Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -58,16 +61,23 @@ export default function useAndroidRippleForView(
         'Unexpected color given for Ripple color',
       );
 
+      let viewProps = {};
```

Comment:
Similar feedback as above about making this `const`. You can also instantiating this after `backgroundValue`:

```
const viewProps = foreground === true
  ? {nativeForegroundAndroid: backgroundValue}
  : {nativeBackgroundAndroid: backgroundValue};
```

Could even return this to line 80 as a ternary property initialization in the return statement.